test_scripts/Defects/7_1/1896_SDL_does_not_send_HeartBeat_request_to_Mobile_HeartBeatTimeout_is_Zero.lua

```diff
@@ -0,0 +1,93 @@
+-- User story: https://github.com/smartdevicelink/sdl_core/issues/1896
+--
+-- Steps to reproduce:
+-- 1. "HeartBeat" param in .ini file is zero
+-- 2. SDL, HMI are running
+-- 3. App is registered.
+-- SDL must:
+-- 1. respond ACK to HeartBeat_request from mobile app
+-- 2. NOT send HeartBeat_request to mobile app
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local common = require("user_modules/sequences/actions")
+local constants = require('protocol_handler/ford_protocol_constants')
+local events = require('events')
+
+--[[ Test Configuration ]]
+config.defaultProtocolVersion = 3
+runner.testSettings.isSelfIncluded = false
+
+--[[ Local Variables ]]
+local appSessionId = 1
+
+local mobSessionConfig = {
+  activateHeartbeat = false,
+  sendHeartbeatToSDL = false,
+  answerHeartbeatFromSDL = true,
```

Comment:
I suppose we need to set `answerHeartbeatFromSDL` to `false` since HB is not expected from SDL